coding-intro/coding-intro-core/coding-intro-introduction/why-learn-to-code.md

```diff
@@ -15,14 +15,17 @@ category: must-know
 Learning how code works isn't only for developers - it can help everyone! 
 
 For example, you'll be able to:
-- write programs that automate processes and save you time (and money!)
-- create websites for your products or projects
-- understand how software works, helping to improve your communication and decision making at work
-- better navigate the modern digital world
-- collaborate with developers better
+- automate everything! save time and money!
+- make a website or a mobile app.
+- collaborate with other developers
+- be your company's software guru.
+- most importantly, control technology so it doesn't control you!
```

Comment:
if we're adding a `.` at the end of each bullet point, we should capitalise the first word:
```suggestion
- Make a website or a mobile app.
- Collaborate with other developers.
- Be your company's software guru.
- Most importantly, control technology so it doesn't control you!
```

Reply:
That was how initially planned on writing them. Changed everything to lower case because of the way the existing content was written.